### PR TITLE
[release-v1.60] Bump tar rpm to fix concurrent host-assisted clone failure

### PR DESCRIPTION
**Pull request by arnongilboa**
**What this PR does / why we need it**:

Manual backport of #3347

The source pvc of clone source-pod `cdi-data-vol` volume is not mounted as `ReadOnly`, and when several host-assisted clones from the same PVC are done concurrently, the `disk.img` ctime may change during the operation, so `tar` fails and the pod is restarted:
```
clone-source.go:41] Subprocess did not execute successfully, result is: '\x01'
/usr/bin/tar: disk.img: file changed as we read it
```

The issue happens with the current tar version 1.34, but seems to be solved in [1.35](https://www.gnu.org/software/tar/):
```
Warn file changed as we read it less often. Formerly, tar warned if the file's size or ctime changed. However, this generated a false positive if tar read a file while another process hard-linked to it, changing its ctime. Now, tar warns if the file's size, mtime, user ID, group ID, or mode changes. Although neither heuristic is perfect, the new one should work better in practice.
```

**Which issue(s) this PR fixes**:

jira-ticket: https://issues.redhat.com/browse/CNV-48778

**Special notes for your reviewer**:

**Release note**:
```release-note
Bump rpms to fix concurrent host-assisted clone tar failure on file change during read
```

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -3345,20 +3345,20 @@ rpm(
...
 rpm(
-    name = "tar-2__1.34-6.el9.aarch64",
-    sha256 = "98a9ca5a25c6aa73b5183b3333abad062a8f82d8b9390d2b2fbdc1eea5b4fb9b",
+    name = "tar-2__1.34-7.el9.aarch64",
+    sha256 = "e3ee12a44a68c84627e43c2512ad8904a4778a82b274d0e8147ca46645f4a1fb",
     urls = [
-        "http://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os/Packages/tar-1.34-6.el9.aarch64.rpm",
-        "https://storage.googleapis.com/builddeps/98a9ca5a25c6aa73b5183b3333abad062a8f82d8b9390d2b2fbdc1eea5b4fb9b",
+        "http://mirror.stream.centos.org/9-stream/BaseOS/aarch64/os/Packages/tar-1.34-7.el9.aarch64.rpm",
+        "https://storage.googleapis.com/builddeps/e3ee12a44a68c84627e43c2512ad8904a4778a82b274d0e8147ca46645f4a1fb",
```
Comment (akalenyu) [inline WORKSPACE:3352]: wondering how you got the mirroring to work. AFAIK this is something that gets added by a bot in such PRs:
https://github.com/kubevirt/containerized-data-importer/pull/3420

Comment (awels) [reply WORKSPACE:3352]: Its probably pointing to the mirroring from the main PR.

Comment (arnongilboa) [reply WORKSPACE:3352]: right

Comment (akalenyu) [reply WORKSPACE:3352]: I actually looked at main, the mirroring was set up in the original PR :O

Comment (arnongilboa) [reply WORKSPACE:3352]: The mirroring would have been automated to the same value by the following `make rpm-deps` anyway. No idea why I kept it as two commits.

Comment (akalenyu) [reply WORKSPACE:3352]: The mirroring isn't done as part of the deps target, it's done by the bot https://github.com/kubevirt/containerized-data-importer/pull/3420
I guess you just predicted the mirror address, and the bot populated it
/approve
/lgtm

**Pull request merged**